samples/stock/standalone-navigator/getrange/demo.js

```diff
@@ -0,0 +1,42 @@
+(async () => {
+    const usdeur = await fetch(
+        'https://www.highcharts.com/samples/data/usdeur.json'
+    ).then(response => response.json());
+
+    const navigator = Highcharts.navigator('navigator-container', {
+        series: [{
+            data: usdeur
+        }]
+    });
+
+    const chart = Highcharts.chart('chart-container', {
+        xAxis: {
+            type: 'datetime'
+        },
+
+        series: [{
+            name: 'USD to EUR',
+            data: usdeur
+        }]
+    });
+
+    navigator.bind(chart);
+
+    document.getElementById('button').addEventListener('click', e => {
+        const { min, max } = navigator.getRange();
+
+        chart.renderer.label(
+            'min: ' + Highcharts.dateFormat('%Y-%m-%d', min) + '<br/>' +
+            'max: ' + Highcharts.dateFormat('%Y-%m-%d', max) + '<br/>',
+            100,
+            100
+        )
+            .attr({
+                fill: '#FCFFC5',
+                zIndex: 8
+            })
+            .add();
+
+        // e.target.disabled = true;
```

Comment:
Do we need this one here or it's a minor development leftover?

If it should stay like that in the demo, then I would suggest adding a short comment explaining why it's here